analysis/src/python/data_analysis/template_errors/template_utils.py

```diff
@@ -0,0 +1,25 @@
+import ast
+from typing import Dict, List
+
+from hyperstyle.src.python.review.application_config import LanguageVersion
+
+from analysis.src.python.data_analysis.utils.code_utils import split_code_to_lines
+from analysis.src.python.evaluation.utils.pandas_utils import get_language_version
+
+
+def get_template_language_version(lang: str) -> LanguageVersion:
+    # TODO: Remove when java17 language version will be added to hyperstyle.
+    if lang == 'java17':
+        return LanguageVersion.JAVA_15
+    return get_language_version(lang)
+
+
+def parse_templates_code(template_config: str) -> Dict[str, List[str]]:
```

Comment:
It seems we already have the similar function - https://github.com/hyperskill/hyperstyle-analyze/blob/b7d148fea82d0599e789d469c69a530edfd9a26f/analysis/src/python/data_analysis/template_errors/search.py#L148